seacrowd/sea_datasets/malaysia_tweets/malaysia_tweets.py

```diff
@@ -0,0 +1,164 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+from seacrowd.utils import schemas
+
+import pandas as pd
+
+# TODO: Add BibTeX citation
+_CITATION = """\
+@InProceedings{10.1007/978-981-16-8515-6_44,
+author="Juan, Sarah Samson
+and Saee, Suhaila
+and Mohamad, Fitri",
+editor="Alfred, Rayner
+and Lim, Yuto",
+title="Social Versus Physical Distancing: Analysis of Public Health Messages at the Start of COVID-19 Outbreak in Malaysia Using Natural Language Processing",
+booktitle="Proceedings of the 8th International Conference on Computational Science and Technology",
+year="2022",
+publisher="Springer Singapore",
+address="Singapore",
+pages="577--589",
+abstract="The study presents an attempt to analyse how social media netizens in Malaysia responded to the calls for ``Social Distancing'' and ``Physical Distancing'' as the newly recommended social norm was introduced to the world as a response to the COVID-19 global pandemic. The pandemic drove a sharp increase in social media platforms' use as a public health communication platform since the first wave of the COVID-19 outbreak in Malaysia in April 2020. We analysed thousands of tweets posted by Malaysians daily between January 2020 and August 2021 to determine public perceptions and interactions patterns. The analysis focused on positive and negative reactions and the interchanges of uses of the recommended terminologies ``social distancing'' and ``physical distancing''. Using linguistic analysis and natural language processing, findings dominantly indicate influences from the multilingual and multicultural values held by Malaysian netizens, as they embrace the concept of distancing as a measure of global public health safety.",
```

Comment:
This line is too long (max line length is 250). 
Run `make check_file=seacrowd/sea_datasets/malaysia_tweets/malaysia_tweets.py` to check the necessary changes to your script.